src/app/shared/components/template/components/task-progress-bar/task-progress-bar.component.scss

```diff
@@ -4,6 +4,7 @@
 // Within the task-card component, the progress bar colour is determined by the status of the task group, i.e.,
 // "notStarted" | "inProgress" | "completed" | "highlighted"
 $progress-colour: var(--task-progress-bar-color, var(--ion-color-primary));
+$progress-wheel-foreground: var(--task-progress-wheel-color, var(--ion-color-secondary-600));
```

Comment:
Progress-wheel-colour abstracted from progress-bar-colour so that it can be set independently in themes.